Docs/sphinx_documentation/source/GPU.rst

```diff
@@ -207,7 +207,7 @@ check the :ref:`table <tab:cmakecudavar>` below.
    +------------------------------+-------------------------------------------------+-------------+-----------------+
    | Variable Name                | Description                                     | Default     | Possible values |
    +==============================+=================================================+=============+=================+
-   | AMReX_CUDA_ARCH              |  CUDA target architecture                       | Auto        | User-defined    |
+   | AMREX_CUDA_ARCH              |  CUDA target architecture                       | Auto        | User-defined    |
```

Comment:
This is not an environment variable, but a CMake option (variable) for AMReX.
CMake options are case sensitive and use the original spelling of AMReX.